1.8/src/main/java/com/superzanti/serversync/ServerSyncRegistry.java

```diff
@@ -0,0 +1,103 @@
+package com.superzanti.serversync;
+
+import java.util.Arrays;
+import java.util.List;
+
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.common.config.Configuration;
+import net.minecraftforge.common.config.Property;
+
+import org.apache.logging.log4j.Logger;
+
+import net.minecraftforge.fml.common.Mod;
```

Comment:
All of these imports in the main mod file for example. These are cpw.mods.fml in 1.7.10 and net.minecraftforge.fml in 1.8. This is just one example, there's many other files that you're using FML things in.